Jenkinsfile

```diff
@@ -147,7 +147,7 @@ pipeline {
       steps {
         dir (env.REPO_DIR) {
           sh '''
-             npx upload \
+             npx --package=@purecloud/web-app-deploy@8 -- upload \
```

Comment:
should this be `web-app-deploy` also ?

Reply:
yes. `@purecloud/web-app-deploy` package has a few programs in its `bin`

From the packages `package.json`
```
  "bin": {
    "upload": "upload.js",
    "deploy": "deploy.js",
    "cached-asset-prefix": "cached-asset-prefix.js",
    "generate-build": "generate-build.js",
    "versioned-url": "versioned-url.js",
    "add-analytics-tags": "add-analytics-tags.js"
  },
```